src/editors/SentryConfigEditor.tsx

```diff
@@ -38,64 +42,80 @@ export const SentryConfigEditor = (props: SentryConfigEditorProps) => {
-          aria-label={ConfigEditorSelectors.SentrySettings.URL.ariaLabel}
-          value={url}
-          onChange={(e) => setURL(e.currentTarget.value)}
-          onBlur={() => onOptionChange('url', url)}
-          width={valueWidth * 2}
-        ></Input>
-      </div>
-      <div className="gf-form" data-testid="sentry-config-editor-org-slug-row">
-        <InlineFormLabel tooltip={ConfigEditorSelectors.SentrySettings.OrgSlug.tooltip} width={labelWidth}>
-          {ConfigEditorSelectors.SentrySettings.OrgSlug.label}
-        </InlineFormLabel>
-        <Input
+        >
+          <Input
+            placeholder={ConfigEditorSelectors.SentrySettings.URL.placeholder}
+            aria-label={ConfigEditorSelectors.SentrySettings.URL.ariaLabel}
+            value={url}
+            onChange={(e) => setURL(e.currentTarget.value)}
+            onBlur={() => onOptionChange('url', url)}
+            width={valueWidth * 2}
+          />
+        </Field>
+        <Field
+          description={ConfigEditorSelectors.SentrySettings.OrgSlug.tooltip}
+          label={ConfigEditorSelectors.SentrySettings.OrgSlug.label}
+          required
+          invalid={!jsonData.orgSlug}
+          error={'Organization is required'}
           data-testid="sentry-config-editor-org-slug"
-          placeholder={ConfigEditorSelectors.SentrySettings.OrgSlug.placeholder}
-          aria-label={ConfigEditorSelectors.SentrySettings.OrgSlug.ariaLabel}
-          value={orgSlug}
-          onChange={(e) => setOrgSlug(e.currentTarget.value)}
-          onBlur={() => onOptionChange('orgSlug', orgSlug)}
-          width={valueWidth * 2}
-        ></Input>
-      </div>
-      <div className="gf-form" data-testid="sentry-config-editor-auth-token-row">
-        <InlineFormLabel tooltip={ConfigEditorSelectors.SentrySettings.AuthToken.tooltip} width={labelWidth}>
-          {ConfigEditorSelectors.SentrySettings.AuthToken.label}
-        </InlineFormLabel>
+        >
+          <Input
+            placeholder={ConfigEditorSelectors.SentrySettings.OrgSlug.placeholder}
+            aria-label={ConfigEditorSelectors.SentrySettings.OrgSlug.ariaLabel}
+            value={orgSlug}
+            onChange={(e) => setOrgSlug(e.currentTarget.value)}
+            onBlur={() => onOptionChange('orgSlug', orgSlug)}
+            width={valueWidth * 2}
+          ></Input>
+        </Field>
         {secureJsonFields?.authToken ? (
-          <>
-            <Input type="text" value="Configured" disabled={true} width={valueWidth * 2}></Input>
-            <Button
-              variant="secondary"
-              className="reset-button"
-              onClick={() => {
-                setAuthToken('');
-                onSecureOptionChange('authToken', authToken, false);
-              }}
-            >
-              {ConfigEditorSelectors.SentrySettings.AuthToken.Reset.label}
-            </Button>
-          </>
+          <Field
```

Comment:
There's a lot going on in this `return` statement, is there a possibility to break some of the inline components that render conditionally into new discrete components?

Reply:
We could potentially break out the secure socks component...